docs/dual-pods.md

```diff
@@ -166,3 +167,36 @@ directs the `vllm serve` process to use the indicated one.
 The name of this Pod combines the name of the relevant node (which is
 presumed to also appear as the value of the hostname label) and the
 set of associated GPUs (hexadecimal rendering of bitmask).
+
+## Sequence Flows
+
+### Inference Server Creation Flow
+
+![dual-pod architecture](./dual-pods-arch.drawio.png)
+
+The key steps for requesting an inference server are:
+
+1. An actor (e.g., a Kubernetes controller or an end-user) creates an inference
+   server requester pod (see example above).
+2. The `inference-server-gpu-allocator` container exposes which accelerators have
+   been allocated on the node and marks itself as ready (i.e., its readiness probe
+   succeeds).
+3. The `inference-server-provider-controller` container watches inference server
+   requester pods (i.e., those with the annotation `dual-pod.llm-d.ai/role: requester`)
+   and delegates lifecycle events (ie. `create`, `update` and `delete`) to
+   the `inference-server-provider` component. Note: multiple implementations of
+   the provider component may exists, but only one is active at any given time.
+4. The `inference-server-provider` implementation selects an inference server that
+   matches the inference server requester pod specification. If no suitable server pod is found,
+   it falls back to the `cold-start provider`.
+5. The `inference-server-provider` sets the `dual-pod.llm-d.ai/inference-server-ip`
```

Comment:
Something was omitted from this sentence. Are we talking about a label or annotation on a Kubernetes object? If so, which one? Are we talking about a file or some content in a file?